tests/vpcpeering_test.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=missing-docstring, line-too-long, protected-access, E1101, C0202, E0602, W0109
+import unittest
+from runner import Runner
+
+
+class TestE2E(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.snippet = """
+
+            provider "aws.source" {
```

Comment:
use the same `alias` syntax as we have in dq-tf-infra/main.tf

Reply:
changed